src/Components/WebView/WebView/src/IpcSender.cs

```diff
@@ -6,22 +6,8 @@
 using Microsoft.AspNetCore.Components.RenderTree;
 using Microsoft.JSInterop;
 
-// Sync vs Async APIs for this.
-// This mainly depends on the underlying support for the browser. Assuming that there is no synchronous API
-// communication is safer, since it's not guaranteed.
-// In that scenario, some APIs need to expose the async nature of the communication. That happens when some
-// component like the renderer needs to know the results of the operation. For example when updating the UI
-// since more code needs to execute afterwards.
-// In other cases like when we try to attach a component to the document, we don't necessarily need to do that
-// since we only care about errors that might happen while attaching the component and the renderer doesn't
-// necessarily need to know about those if we are terminating the component/host as a result.
-// If we decide we need to expose the async nature of the communication channel, then we will need to keep track
-// of all the message pairs/completions across the IPC channel.
 namespace Microsoft.AspNetCore.Components.WebView
 {
-    // These are all the messages .NET needs to know how to dispatch to JS
-    // TODO: Proper serialization, error handling, etc.
```

Comment:
The serialization is already adequate. Error handling is something we're tightening up as part of implementing unit tests.